.github/workflows/bash.yml

```diff
@@ -46,3 +46,10 @@ jobs:
             tests/functional/shutdown/09-now2.t \
             tests/functional/shutdown/13-no-port-file-check.t \
             tests/functional/shutdown/14-no-dir-check.t
+
+      - name: Upload
+        if: failure()
+        uses: actions/upload-artifact@v2
+        with:
+          name: cylc-run
+          path: cylc-run
```

Comment:
@hjoliver I think `path` is relative to where the action is running. In the log lines next to the error uploading file in the last run, we have

```
Test Summary Report
-------------------
tests/functional/broadcast/00-simple.t              (Wstat: 0 Tests: 5 Failed: 4)
  Failed tests:  2-5
```

I think `tests/functional/broadcast/00-simple.t` is relative to the working directory of the action. So the `cylc-run` would be probably somewhere like `/root/cylc-run/` (from the error log line `suite logs can be found under: /root/cylc-run/cylctb-20201112T101740Z/functional/broadcast/00-simple/log/suite/`).

WDYT?

Reply:
Ah yes, this functionality hadn't been tested, yet.

Yes, my bad, looks like it's a relative path so `~/cylc-run`, `/root/cylc-run` or possibly `../cylc-run`.